docs/dev/table/connectors/elasticsearch.md

```diff
@@ -126,6 +126,20 @@ Connector Options
       <td>String</td>
       <td>Delimiter for composite keys ("_" by default), e.g., "$" would result in IDs "KEY1$KEY2$KEY3"."</td>
     </tr>
+    <tr>
+      <td><h5>username</h5></td>
+      <td>optional</td>
+      <td style="word-wrap: break-word;">(none)</td>
+      <td>String</td>
+      <td>Username used to connect to Elasticsearch instance.</td>
```

Comment:
What do you think about adding more explanation about ES authentication here? I guess not all the users know the auth feature of Elasticsearch. For example:

```
Elasticsearch doesn't pre-bundled security feature, but you can enable it by following the [guideline](https://www.elastic.co/guide/en/elasticsearch/reference/7.x/configuring-security.html) to secure an Elasticsearch cluster.
```